content/trusted-content/official-images/using.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Using Docker Official Images
+description: |
+  Learn about building applications with Docker Official images
+  and how to interpret the tag names they use.
+keywords: docker official images, doi, tags, slim, feedback, troubleshooting
+weight: 10
+---
+
+Docker recommends you use the Docker Official Images in your projects.
+These images have clear documentation, promote best practices, and are regularly updated.
+Docker Official Images support most common use cases, making them perfect for new Docker users.
+Advanced users can benefit from more specialized image variants as well as review Docker Official Images as part of your `Dockerfile` learning process.
+
+## Tags
+
+The repository description for each Docker Official Image contains a
+**Supported tags and respective Dockerfile links** section that lists all the
+current tags with links to the Dockerfiles that created the image with those
+tags. The purpose of this section is to show what image variants are available.
+
+![Example: supported tags for Ubuntu](../images/supported_tags.webp)
+
+Tags listed on the same line all refer to the same underlying image. Multiple
+tags can point to the same image. For example, in the previous screenshot taken
+from the `ubuntu` Docker Official Images repository, the tags `24.04`,
+`noble-20240225`, `noble`, and `devel` all refer to the same image.
+
+The `latest` tag for a Docker Official Image is often optimized for ease of use
+and includes a wide variety of useful software, such as developer and build tools.
+Because of their ease of use and wide applicability, `latest` images are often
+used in getting-started guides, but they're typically not optimized for production use.
+
+## Slim images
+
+A number of operating system and language stacks such as
+[Debian](https://hub.docker.com/_/debian/),
+[Node.js](https://hub.docker.com/_/node/),
+[Python](https://hub.docker.com/_/python/), and
+[Ruby](https://hub.docker.com/_/ruby/) have `slim` tag variants
+designed to provide a lightweight, production-ready base image
+with fewer packages.
+
+A typical consumption pattern for `slim`
+images is as the base image for the final stage of a
+[multi-staged build](https://docs.docker.com/build/building/multi-stage/).
+For example, you build your application in the first stage of the build
+using the `latest` variant and then copy your application into the final
+stage based upon the `slim` variant. Here is an example `Dockerfile`.
+
+```dockerfile
+FROM node:latest AS build
+WORKDIR /app
+COPY package.json package-lock.json ./
+RUN npm ci
+COPY . ./
+FROM node:slim
+WORKDIR /app
+COPY --from=build /app /app
+CMD ["node", "app.js"]
+```
+
+## Alpine
```

Comment:
Should these following subheadings have the "images" suffix like "Slim images" does above, or should it not have the suffix?  (It feels like for consistency, we should probably do one or the other)

Reply:
sure, let me add `images`